### DISCUSSION TITLE
Fedora, set R path

### DISCUSSION THREAD

**Discussion by iztokstotl** [microsoft/vscode-discussions#894] [Extension Development QnA]
I have set up r term path to or /usr/bin or /usr/bin/R on VS code and Fedora 39 in R extension vor VS setup window.

I tried different combinations for workspace and user setting and I get those errors:

![image](https://github.com/microsoft/vscode-discussions/assets/11088416/d087c4e3-c859-436a-bf81-255f14820678)

Any ideas how to proceed ?

**Comment by olguzzar** [comment]
Hi @iztokstotl, if you haven't already, you should check out Stack Overflow for help with this issue as there are a lot of similar posts like [this one](https://stackoverflow.com/questions/65823681/setting-up-visual-studio-code-to-work-with-r-win32-cant-use-r). You might need to update the path in other places, or perhaps you need to use the full path, not just "/usr/bin".

This forum is for extension authors, so I'm going to close this discussion. Thanks!
